apple/swift-docc
# Filter out topics that aren't available in the page's language

Bug/issue #, if applicable: rdar://89124291

## Summary

Filters out topics that aren't available in the language of the symbol that curates them. Note that this was already working for automatically curated topics, but not manually curated topics.

This PR also includes making tutorial content available on all the languages in which the framework is available (last commit),  

Also, since the original fix of this PR filters out topics that aren't available in the parent page's language, it was also causing DocC to filter out tutorial content from Objective-C pages. As such, the last commit of this PR resolves this issue by making tutorials multi-language as well, if the framework it's documenting is multi-language.

## Dependencies

None.

## Testing

1. Create symbol graph files with symbols that are only available in Swift and symbols only available in Objective-C, and curate them manually in a documentation extension file or in documentation comments.
2. Verify that the rendered documentation doesn't show the Objective-C–only symbols when browsing the Swift documentation, and vice-versa.

## Checklist


## TODO

- [x] Make Tutorials multi-language so that they can be curated in Objective-C pages. They're currently hardcoded to be Swift-only
